recipes/xvega/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xvega" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "395d2c6890a4d9e28764ab39569663647138a7a9430f37c08fedd04c2cbd2dff" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xvega/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+  host:
+    - nlohmann_json
+    - xproperty
+    - xtl
+  run:
+    - nlohmann_json
+    - xproperty
+    - xtl
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xvega  # [unix]
+    - test -d ${PREFIX}/include/xvega/base  # [unix]
+    - test -d ${PREFIX}/include/xvega/functions  # [unix]
+    - test -d ${PREFIX}/include/xvega/grammar  # [unix]
+    - test -d ${PREFIX}/include/xvega/utils  # [unix]
+    - test -f ${PREFIX}/include/xvega/xvega.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/base/xvega-base.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/functions/populate_data.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/grammar/marks.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/utils/serialize.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xvega/xvegaConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xvega/xvegaConfigVersion.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\xvega\xvega.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\base\xvega-base.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\functions\populate_data.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\grammar\marks.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\utils\serialize.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xvegaConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xvegaConfigVersion.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/QuantStack/xvega
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: A C++ backend for vega-lite
+  description: vega-lite visualization system for the xeus-cling kernel
+  dev_url: https://github.com/QuantStack/xvega
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
```

Comment:
@SylvainCorlay are you willing to be a maintainer?

Reply:
+1